lang/cs/Org.Apache.REEF.Client/API/IREEFClient.cs

```diff
@@ -43,5 +44,12 @@ public interface IREEFClient
         /// <returns>IDriverHttpEndpoint</returns>
         [Unstable("0.13", "Working in progress for what to return after submit")]
         IDriverHttpEndpoint SubmitAndGetDriverUrl(IJobSubmission jobSubmission);
+
+        /// <summary>
+        /// Returns the Application status in running the job
+        /// </summary>
+        /// <returns></returns>
+        [Unstable("0.14", "Working in progress for rest API status returned")]
+        Task<ApplicationState> GetJobStatus();
```

Comment:
For which job are we getting the status? Until now, `IREEFClient` supported multiple submissions. Wouldn't it be easier to return this as part of one of the `Submit...` methods?